pkg/workloadmeta/store.go

```diff
@@ -148,7 +142,11 @@ func (s *store) Start(ctx context.Context) {
 		}
 	}()
 
-	s.startCandidates(ctx)
+	go func() {
```

Comment:
If I understand now the `startCandidates` is made async (in another goroutine). 

do we have any dropback?